src/algebraic_geometry/presheafed_space.lean

```diff
@@ -117,17 +89,13 @@ lemma id_c (X : PresheafedSpace.{v} C) :
 lemma comp_c {X Y Z : PresheafedSpace.{v} C} (α : X ⟶ Y) (β : Y ⟶ Z) :
   (α ≫ β).c = (β.c ≫ (whisker_left (opens.map β.f).op α.c)) := rfl
 @[simp] lemma id_c_app (X : PresheafedSpace.{v} C) (U) :
-  ((𝟙 X) : X ⟶ X).c.app U = (eq_to_hom (by { dsimp, op_induction U, cases U, refl })) :=
+  ((𝟙 X) : X ⟶ X).c.app U = eq_to_hom (by tidy) :=
 begin
   simp only [id_c],
-  op_induction U,
-  cases U,
-  dsimp,
-  simp,
+  tidy,
```

Comment:
Should this proof now become `by delta id_c; tidy` instead of the begin-end block?
(Really exciting to see all these proofs vanish, btw!)

Reply:
Changed to a `by`.